lang/ar.css

```diff
@@ -0,0 +1,5 @@
+:lang(ar) .theorem::before { content: 'مبرهنة ' counter(theorem) '. ' !important; }
+:lang(ar) .lemma::before { content: 'تمهيدية ' counter(theorem) '. ' !important; }
+:lang(ar) .proof::before { content: 'البرهان. ' attr(title) !important; }
+:lang(ar) .definition::before { content: 'تعريف ' counter(definition) '. ' !important; }
+body{direction:rtl}pre,code{direction:ltr}th,td{text-align:right}nav ol{padding-right:2rem}nav li:before{padding-left:.85rem}.proof:after{left:-12px}.footnotes p{text-align:right}.latex span:nth-child(1){margin-right:-0.48em;margin-left:-0.15em}.latex span:nth-child(2){margin-right:-0.1667em;margin-left:-0.125em}
```

Comment:
Sorry for not getting back to this in a long time. Could you please prettify line 5. Also, why is the LaTeX class changed? Does this have to do something with RTL because I only see a minor difference?

![image](https://user-images.githubusercontent.com/24668338/90310559-d67a1b80-def2-11ea-933d-265575ed0115.png)
Top: your change
Bottom: current
![image](https://user-images.githubusercontent.com/24668338/90310564-dd089300-def2-11ea-980f-b791faf5710d.png)

Reply:
and please add `right: initial;` to .proof::after in order to overwrite the value set by the default stylesheet